### PR TITLE
Fix yast2_i test

### PR DESCRIPTION
**Pull request by kravciak**
If yast2-sw_shows_summary is not matched then test skips alt-f command
but expects yast to finish. Default timeout should help.

Also removed soft_fail, as it depends only on static variable from test
setup and it produces soft_fail where it should not.

Fix for: https://openqa.suse.de/tests/600708#step/yast2_i/17
Invalid soft_fail: https://openqa.opensuse.org/tests/272298#step/yast2_i/16
Local runs:
http://dhcp91.suse.cz/tests/2698
http://dhcp91.suse.cz/tests/2697

### CODE REVIEW

**Comment by coolo** [comment]
IMO: this is not testing the functionality of that module any longer. If yast doesn't show the summary, this means the package is chosen badly or the system not prepared well enough to test.

**Comment by kravciak** [comment]
fixed

File: tests/console/yast2_i.pm [hunk 1]
```diff
@@ -81,20 +81,13 @@ sub run() {
         send_key 'alt-o';
     }
 
-    # Whether summary is shown depends on PKGMGR_ACTION_AT_EXIT in /etc/sysconfig/yast2
```
Comment (coolo) [inline tests/console/yast2_i.pm]: do you see the comment you just removed? If updating the system, the sysconfig will tell yast not to show a summary. *That* is the reason why we have this variable, so we can set this in upgrade cases. If it's set in the wrong test suites, we should fix that.

Comment (coolo) [inline tests/console/yast2_i.pm]: but I'm not sure we're actually testing upgrades of such a system. sle11 and 11.X will have this set. Not sure when the default was changed.

**Comment by kravciak** [comment]
I found only one update test that does not show summary
 - https://openqa.opensuse.org/tests/275965#step/yast2_i/12

Variable is set in many update tests, maybe leftover or I missed something (checked mostly Leap42.2 & TW)
Does this look better?

**Comment by coolo** [comment]
Yes, clean up these variables.

**Comment by coolo** [comment]
And so I agree to the change

File: products/opensuse/main.pm [hunk 2]
```diff
@@ -105,9 +105,6 @@ if (check_var('DESKTOP', 'minimalx') || get_var('DESKTOP_MINIMALX_INSTONLY')) {
 set_var('LEAP', get_var('VERSION', '') =~ /(?:[4-9][0-9]|[0-9]{3,})\.[0-9]/);
 set_var("PACKAGETOINSTALL", "xdelta");
 set_var("WALLPAPER",        '/usr/share/wallpapers/openSUSEdefault/contents/images/1280x1024.jpg');
-if (!defined get_var("YAST_SW_NO_SUMMARY")) {
```
Comment (kravciak) [inline products/opensuse/main.pm]: Not used anymore if code from registration.pm is removed

**Comment by coolo** [comment]
when we care for sle11 (again) we will have to come up with something - but for now its reasonable cleanup.

**Pull request merged**